### PR TITLE
Feature/javascript

### PR DESCRIPTION
**Pull request by santanam23**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -41,9 +41,9 @@
     "prettier": "^2.5.1",
     "sequelize": "^6.15.1",
     "socket.io": "^4.4.1",
-    "typescript": "^4.5.5",
     "ts-node": "^10.5.0",
-    "tslib": "^2.3.1"
+    "tslib": "^2.3.1",
```
Comment (moonryc) [inline package.json:45]: This change is unneeded

File: views/signup.handlebars [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  <div>
+    <label for="password-signup">password:</label>
+    <input type="password" id="password-signup" />
+  </div>
+  <div>
+    <button type="submit">signup</button>
+  </div>
+</form>
+
+<script src="/javascript/login.js"></script>
```
Comment (moonryc) [inline views/signup.handlebars:21]: This should link to signup.js, signup. js does not exist

File: views/homepage.handlebars [hunk 3]
```diff
@@ -1,15 +1,16 @@
...
+<div class="sub-section">
+  <h2>What We Can Help You With</h2>
+</div>
+
+<div class="row">
+  <div class="column" style="background-color:rgb(192, 200, 191);">Javascript</div>
+  <div class="column" style="background-color:rgb(192, 200, 191);">HTML</div>
+  <div class="column" style="background-color:rgb(192, 200, 191);">CSS</div>
+</div>
+
```
Comment (moonryc) [inline views/homepage.handlebars:15]: There need to be more than this, describe how to use the website

File: views/layouts/main.handlebars [hunk 4]
```diff
@@ -13,20 +13,21 @@
     <header>
       <h1>
         <a href="/">Teach Bot</a>
+        <a href="/signup">Sign Up</a>
```
Comment (moonryc) [inline views/layouts/main.handlebars:16]: This needs to be in the {{else}} statement below

File: package-lock.json [hunk 5]
```diff
@@ -0,0 +1,13900 @@
+{
```
Comment (moonryc) [inline package-lock.json:1]: revert to your old package-lock.json or pull from main/develop, re adding the entire file is pointless

**Review by moonryc — CHANGES_REQUESTED** [review]
Rejected

File: package.json (hunk 1 above)
Comment (santanam23) [reply package.json:45]: meant to delete socket.io

File: views/homepage.handlebars (hunk 3 above)
Comment (santanam23) [reply views/homepage.handlebars:24]: added more but not finalized

**Pull request closed without merging**